keylime/tpm/tpm_main.py

```diff
@@ -68,7 +68,14 @@ def __get_tpm2_tools(self):
         # Extract the full semver release number.
         self.tools_version = version_str.split("-")
 
-        if Version(self.tools_version[0]) >= Version("4.2"):
+        if Version(self.tools_version[0]) >= Version("5.4") or (
+            Version(self.tools_version[0]) == Version("5.3")
+            and len(self.tools_version) > 1
+            and int(self.tools_version[1]) >= 24
```

Comment:
why so specific? is there a patched version of 5.3 that backport the feature in an specific distro? If so a comment would help.

Reply:
When you build from git this is the first version that introduces that change. Because the change is only 5 days old, I don't think any distro has backported that yet.

I'll add a comment.

Reply:
Would make sense then to leave it to 5.4 only.  The other option is to unconditionally do the scan and pay the price.

Reply:
Because of https://github.com/tpm2-software/tpm2-tools/pull/3144 you currently need to run a git snapshot if you want to attest a modern system that uses the latest Shim.

I still want to run the scan only on versions that require it, so my idea would be to keep the current check and then remove it once 5.4 is released with the necessary fixes.